default-grimoirelab-settings/organizations.json

```diff
@@ -10618,6 +10900,330 @@
                 "domain": "epfl.ch",
                 "is_top": true
             }
+        ],
+        "北京大学": [
```

Comment:
You need to change it to English name, also the following Chinese names.

Reply:
done